api/src/main/java/org/apache/iceberg/metrics/ScanReport.java

```diff
@@ -226,6 +236,16 @@ public Counter skippedDeleteFiles() {
       return metricsContext().counter(SKIPPED_DELETE_FILES, MetricsContext.Unit.COUNT);
     }
 
+    @Value.Derived
+    public Counter scannedDeleteManifests() {
+      return metricsContext().counter(SCANNED_DELETE_MANIFESTS, MetricsContext.Unit.COUNT);
```

Comment:
Maybe we should make `COUNT` the default

Reply:
I created https://github.com/apache/iceberg/pull/5912 to do this